packages/docs/community/release-process/smart-contracts.md

```diff
@@ -133,6 +143,21 @@ All releases should be evaluated according to the following tests.
 
 All changes since the last release should be covered by unit tests. Unit test coverage should be enforced by automated checks run on every commit.
 
+### Manual Checklist
+
+After a successful release execution on a testnet, the resulting network state should be spot-checked to ensure that no regressions have been caused by the release. Flows to test include:
+
+- Do a cUSD and CELO transfer
```

Comment:
could have a bash snippet under each bullet point:
```bash
celocli transfer:dollars --from=<addr> --value=<number> --to=<addr>
celocli transfer:celo --from=<addr> --value=<number> --to=<addr>
```

Reply:
Great idea!

Reply:
I think we can expand on these later.